source/request-as-event-emitter.ts

```diff
@@ -130,7 +130,12 @@ export default (options: NormalizedOptions, input?: TransformStream) => {
 
 				const rawCookies = typedResponse.headers['set-cookie'];
 				if (options.cookieJar && rawCookies) {
-					await Promise.all(rawCookies.map((rawCookie: string) => setCookie!(rawCookie, typedResponse.url!)));
+					let promises = rawCookies.map((rawCookie: string) => setCookie!(rawCookie, typedResponse.url!));
+					if (options.ignoreInvalidCookies) {
+						promises = promises.map(p => p.catch(_ => { /* ignore */ }));
```

Comment:
```suggestion
						promises = promises.map(p => p.catch(() => {}));
```

Reply:
The comment inside the brackets is not needed.

Reply:
Fixed it.